adr/choose_npm_publish.md

```diff
@@ -0,0 +1,42 @@
+---
+category: adrs
+expires: 2020-11-05
+---
+
+# Choose NPM Publish
+
+## Status
+
+Accepted
+
+## Context
+
+As we split our code in microservices on the front-end, we need to have a way of sharing common code across products.
```

Comment:
Microservices on the front-end or micro-front-ends? Maybe also specify front-end/JS code across products?

Reply:
Change this sentence now to be more specific to our current usage with examples too.